ChartboostMediationDemo/build.gradle

```diff
@@ -31,37 +31,34 @@ android {
 
 dependencies {
     // Chartboost Mediation
-    implementation 'com.chartboost:chartboost-mediation-sdk:4.0.0'
-
-    // Chartboost Mediation requirement
-    implementation 'org.greenrobot:eventbus:3.3.1'
+    implementation 'com.chartboost:chartboost-mediation-sdk:4.1.0'
 
     // Chartboost Mediation Adapters
     implementation 'com.chartboost:chartboost-mediation-adapter-adcolony:4.4.8.0.0'
-    implementation 'com.chartboost:chartboost-mediation-adapter-admob:4.21.3.0.0'
-    implementation 'com.chartboost:chartboost-mediation-adapter-applovin:4.11.5.5.0'
+    implementation 'com.chartboost:chartboost-mediation-adapter-admob:4.21.5.0.1'
+    implementation 'com.chartboost:chartboost-mediation-adapter-applovin:4.11.8.1.0'
     implementation 'com.chartboost:chartboost-mediation-adapter-chartboost:4.9.2.0.0'
-    implementation 'com.chartboost:chartboost-mediation-adapter-amazon-publisher-services:4.9.6.2.0'
+    implementation 'com.chartboost:chartboost-mediation-adapter-amazon-publisher-services:4.9.7.0.1'
     implementation 'com.chartboost:chartboost-mediation-adapter-digital-turbine-exchange:4.8.2.1.0'
-    implementation 'com.chartboost:chartboost-mediation-adapter-google-bidding:4.21.3.0.0'
+    implementation 'com.chartboost:chartboost-mediation-adapter-google-bidding:4.21.5.0.1'
```

Comment:
let's also update all the other networks

Reply:
Oh, neat. Will do.